### DISCUSSION TITLE
Imagenet1k is too big!

### DISCUSSION THREAD

**Discussion by mrFocusXin** [open-mmlab/mmcv#2617] [Q&A]
**As title, The Imagenet1k is too big to me, my machine can't work with Imagenet1k .  But i still want to run this program, how should i do ? I edit th batch size smaller,but it still be killed because excessive memory usage. Does anyone have any ideas or experience? Your reply will be very helpful to me!**

![image](https://user-images.githubusercontent.com/41821856/220852627-92d77317-6bce-4fed-a6e6-791ac97e09e5.png)

**Comment by mrFocusXin** [comment]
I have fixed it.

**Reply by C1rN09** [reply]
Would you share what problem you have encountered and how you fixed it? This will be helpful to others ^_^

**Reply by mrFocusXin** [reply]
No problem. The ImageNet1k dataset is too large to run for me, and i change my own dataset format to ImageNet1k-style, it works finally!
